api/v1alpha8/openstackmachine_types.go

```diff
@@ -45,13 +45,9 @@ type OpenStackMachineSpec struct {
 	// The flavor reference for the flavor for your server instance.
 	Flavor string `json:"flavor"`
 
-	// The name of the image to use for your server instance.
+	// The image to use for your server instance.
 	// If the RootVolume is specified, this will be ignored and use rootVolume directly.
```

Comment:
If I understand correctly, this is no longer true: we will now use the image requested by the filter when creating the root volume? If so, how about

```suggestion
	// If the rootVolume is specified, this be used when creating the root volume.
```

Reply:
I'm not sure why we had this comment before, since the image ID was not ignored and used when creating the rootVolume:
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/98f154bafb2b51318a7d3eb6655a206a491c9620/pkg/cloud/services/compute/instance.go#L451

So your suggestion is valid now is also valid without this change. I'll accept it.